integration_test/tests/util.rs

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! Tests for methods found under the `== Util ==` section of the API docs.
+
+#![allow(non_snake_case)] // Test names intentionally use double underscore.
+#![allow(unused_imports)] // Because of feature gated tests.
+
+use bitcoin::hex::FromHex as _;
+use bitcoin::{Amount};
+use integration_test::{Node, NodeExt as _, Wallet};
+use node::vtype::*; // All the version specific types.
+
+#[test]
+fn utils__createmultisig__modelled() {
```

Comment:
```suggestion
fn utils__create_multisig__modelled() {
```

I tried to use Rust idiomatic names (excluding the double underscore) instead of Core names (all squashed together). I tried to find a balance of readable identifiers and uniformity with Core. I guess I messed up by naming all the `client` macros how they are - in hindsighte they should have underscores in them.